e2e/runAllTests.js

```diff
@@ -0,0 +1,63 @@
+import { execSync } from 'node:child_process';
+import fs from 'node:fs/promises';
+import path from 'node:path';
+import process from 'node:process';
+
+/**
+ * Run All Tests: This script executes the lint command on all subfolders under `fixtures`, in order
+ * to validate the correctness of our plugin.  Each fixture installs the *built* package.  So this should
+ * only be run after a build has been done.
```

Comment:
Out of curiosity, how is the performance impact of this on our builds?

Reply:
In total, it adds a little under a minute to the overall CI runtime. Mainly because the E2E steps, have to wait for the build to complete.  Just looking at the build readout below, it took 38s for the build job, which ran in parallel to all of the other existing jobs.  At that point all the E2E jobs kicked off in parallel, and the longest running of those are the windows ones, which took about a minute each (the ubuntu ones took between 40-50s).

Reply:
RE: performance, though, there will be a roughly linear impact to the length of those steps, for each e2e test that's added in the future.  So, one thing to consider, if many more are added, would be to run some number of them concurrently.  I wouldn't necessarily go to that right now, because it adds some amount of complexity.  With just two tests, there's not much to gain with doing them concurrently.  But if there were 10 or more, it might start to make sense.